CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.4.5]
+
+**Improvement**
```

Comment:
Shouldn't it say feature instead?

Reply:
I've followed the change according to this documentation
https://keepachangelog.com/en/1.0.0/#how